app/models/user.rb

```diff
@@ -10,7 +10,7 @@ def self.member(id)
     new(id: id)
   end
 
-  aasm do
+  aasm whiny_transitions: false do
```

Comment:
whiny? 😸

Reply:
Haha, it helps not throw up too much when an invalid state change is made - like when they visit a bookmarked page that showed up when they were on a different state.

Reply:
Ah interesting - this stops the `invalid transition` error from being raised. Wished I had seen that earlier